Makefile

```diff
@@ -7,12 +7,12 @@ ifeq ($(shell go help mod >/dev/null 2>&1 && echo true), true)
 	GO_BUILD=GO111MODULE=on $(GO) build -mod=vendor
 endif
 DESTDIR ?=
-PREFIX ?= /usr
+PREFIX ?= /usr/local
```

Comment:
This change is breaking tests as Podman is configured to look at `/usr/libexec`

Reply:
We could either set `PREFIX=/usr` in the tests during installation or use a custom containers.conf here. Setting the prefix might be the faster one :)

Reply:
hmm, wouldn't `/usr/local` be the sane default for a `make && make install` ? If you prefer `/usr`, is it ok if we have the same PREFIX in the docs/ dir as well?

Reply:
err, maybe I misread your comment ...

Reply:
> We could either set `PREFIX=/usr` in the tests during installation or use a custom containers.conf here. Setting the prefix might be the faster one :)

where exactly does this test live? I guess somewhere in the test/ dir, or is it something separate in the CI?

Reply:
Setting it here https://github.com/containers/oci-seccomp-bpf-hook/blob/master/.cirrus.yml#L128 should work.

Reply:
We're using Cirrus CI.

Reply:
cool, I'll update the PR with a change for that, thanks.

Reply:
updated and squashed, thanks.